### DISCUSSION TITLE
Pin Layout mistake

### DISCUSSION THREAD

**Discussion by shaposhnikow** [omersiar/esp-rfid#529] [General]
I connected the RDM6300 reader as indicated in the table - nothing worked. Found an error: instead of D5 (GPIO-14) TX RDM6300 must be connected to RXD0 (GPIO-03). So everything works correctly.

**Comment by matjack1** [comment]
hey @shaposhnikow thank you for the feedback! I've actually noticed that there's no setting for the RDM6300 for real :)

I'm making this into an issue so that we can double check this and at least update the docs.
